server/src/db/Seeder.js

```diff
@@ -0,0 +1,14 @@
+import { connection } from "/Users/rjfox1992/challenges/readingListApp/breakable-toy-launch31/server/src/boot.js";
```

Comment:
you shouldn't use this absolute path here, this will break it on other machines. instead use 
```
import { connection } from "../boot.js"
```

Reply:
updated